dailyjournal.sql

```diff
@@ -4,21 +4,21 @@ CREATE TABLE `Entries` (
   `entry` TEXT NOT NULL,
   `date` INTEGER NOT NULL,
   `moodId` INTEGER NOT NULL,
-  FOREIGN KEY(`moodId`) REFERENCES `Moods`(`id`),
+  FOREIGN KEY(`moodId`) REFERENCES `Moods`(`id`)
 );
 
 CREATE TABLE `Moods` (
   `id`	INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
   `label` TEXT NOT NULL,
 );
 
-INSERT INTO `Entries` VALUES(null, "12345", "123", 1598458543321, 1)
-INSERT INTO `Entries` VALUES(null, "abc", "123", 1598458548239, 2, 2)
-INSERT INTO `Entries` VALUES(null, "Delete", "Now Deleting", 1598458559152, 1, 3)
-INSERT INTO `Entries` VALUES(null, "ANGRY", "jij", 1598557358781, 3, 4)
-INSERT INTO `Entries` VALUES(null, "678", "Now Deleting", 1598557373697, 4, 5)
+INSERT INTO `Entries` VALUES (null, "12345", "123", 1598458543321, 1);
+INSERT INTO `Entries` VALUES(null, "abc", "123", 1598458548239, 2, 2);
+INSERT INTO `Entries` VALUES(null, "Delete", "Now Deleting", 1598458559152, 1, 3);
+INSERT INTO `Entries` VALUES(null, "ANGRY", "jij", 1598557358781, 3, 4);
+INSERT INTO `Entries` VALUES(null, "678", "Now Deleting", 1598557373697, 4, 5);
```

Comment:
Also, you have extra values in your last four INSERT statements for the `Entries` table here.

A correct statement would look like:
```INSERT INTO `Entries` VALUES("abc", "123", 1598458548239, 2);```

i.e. VALUES should be: `(concept, entry, date, moodId)`